CocoaLumberjack.podspec

```diff
@@ -28,13 +27,14 @@ Pod::Spec.new do |s|
   s.default_subspecs = 'Core'
 
   s.subspec 'Core' do |ss|
-    ss.source_files         = 'Classes/CocoaLumberjack.h', 'Classes/DD*.{h,m}', 'Classes/Extensions/*.{h,m}', 'Classes/CLI/*.{h,m}'
-    ss.private_header_files = 'Classes/DD*Internal.{h}'
+    ss.source_files         = 'Sources/CocoaLumberjack/**/*.{h,m}'
+    ss.private_header_files = 'Sources/CocoaLumberjack/DD*Internal.{h}'
+    ss.public_header_files  = 'Sources/CocoaLumberjack/include/*.{h}', 'Sources/CocoaLumberjack/Supporting Files/*.{h}'
```

Comment:
Explicitly defining public header files isn't necessary. CocoaPods treats all headers as public unless they're present in `private_header_files`